Rebranding to GoPlay

michaelarnauts opened the pull request:
* Fixes for rebranding to GoPlay
* Disabled categories for now, since they don't seem to be used anymore

Download zips:
[plugin.video.viervijfzes-0.3.1-goplay-8de921a.zip](https://github.com/add-ons/plugin.video.viervijfzes/files/5889776/plugin.video.viervijfzes-0.3.1-goplay-8de921a.zip)
[plugin.video.viervijfzes-0.3.1+matrix.1-goplay-8de921a.zip](https://github.com/add-ons/plugin.video.viervijfzes/files/5889777/plugin.video.viervijfzes-0.3.1%2Bmatrix.1-goplay-8de921a.zip)

timrijckaert commented on resources/lib/viervijfzes/__init__.py:
```diff
@@ -37,14 +37,30 @@
-        iptv_id='zes.be',
+        iptv_id='play6.be',
+        youtube=[],
+    )),
```
Play 7? 😲

frankdpGH commented:
Thanks for this remediation, working on my side.

wputman29 commented:
@michaelarnauts , last time I checked yesterday, the tv guide didn't sync (yet) and it returned me an error when I refreshed the channels and guides in IPTV Manager, can you confirm that too?

michaelarnauts commented:
@wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.

https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)

wputman29 commented:
> @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> 
> https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)

Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?

michaelarnauts commented:
> > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> 
> @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> 
> Reset of my kodi did the job. Thanks anyway ;)

I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

wputman29 commented:
> > > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> > 
> > 
> > @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> > Reset of my kodi did the job. Thanks anyway ;)
> 
> I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

When clearing Kodi completely from scratch, it works fine with the new logo, but from the moment you import 'old' userdata, even when the app is not installed yet, the old logo comes back once you install it. I even tried removing all the userdata from the viervijfzes addon, it still happens. A clean wipe of all userdata did it, now it works fine with the new logo. Let's hope they keep it for a while ;)

MPParsley commented:
Thank you very much for this update!

Not sure if this is related but I'm still experiencing some playback issues. I'm not seeing any indication of previously played items and somehow I can play all episodes from Masterchef S06 up until episode 12. As of 13 and up the video can't be played.

Tried both zip files mentioned in this PR and uninstalled/reinstalled the app.

dagwieers commented:
> I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

In the past we found that only when the add-on was updated in the Kodi repository, the add-on would reliably be changed in Kodi everywhere.

MPParsley commented on resources/lib/modules/channels.py:
```diff
@@ -7,10 +7,9 @@
 from resources.lib.viervijfzes import CHANNELS, STREAM_DICT
 from resources.lib.viervijfzes.auth import AuthApi
-from resources.lib.viervijfzes.content import CACHE_AUTO, CACHE_ONLY, ContentApi
+from resources.lib.viervijfzes.content import ContentApi
```
Was this done intentionally?

vermope commented:
@wputman29 When updating the metadata from the settings menu all logos are renewed and visible.

michaelarnauts commented:
> > > > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > > > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> > > 
> > > 
> > > @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> > > Reset of my kodi did the job. Thanks anyway ;)
> > 
> > 
> > I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?
> 
> When clearing Kodi completely from scratch, it works fine with the new logo, but from the moment you import userdata, even when the app is not installed yet, the old logo comes back once you install it. I even tried removing all the userdata from the viervijfzes addon, it still happens. Is it possible IPTV manager caches that logo too, it's the only addon that I can think of that would use the data of it in order to send it to the PVR for the tv guide? Only a clean wipe of all userdata did it in the end, now it works fine with the new logo. Let's hope they keep it for a while ;)

I've also had this on my development machine, and I fixed it there by removing the Textures.db file in the Kodi userdata/database folder while Kodi wasn't running.

wputman29 commented:
Thanks for checking 😊
